vsintegration/Vsix/VisualFSharpFull/VisualFSharpFull.csproj

```diff
@@ -43,7 +43,7 @@
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
-    <VSRootSuffix>FSharpDev</VSRootSuffix>
+    <VSRootSuffix>RoslynDev</VSRootSuffix>
```

Comment:
Whoa this is wrong! Or we could merge FSharpDev with RoslynDev? I'm not massively fussed - it will help with cross-project dev.